### ISSUE TITLE
My enclosure designs made for the Faketec

### ISSUE THREAD

**Issue by lupusworax** [gargomoma/fakeTec_pcb#8]
I just wanna link some of my models I made lately that were made for the Faketec. 

The "FAKECAP": 
https://www.printables.com/model/1067553-meshtastic-fakecap-super-slim-nrf52-gps-node

The "Tamameshi": 
https://www.printables.com/model/1061194-lwc-meshtastic-tamameshi-portable-faketec-node

The " Meshformer":
https://www.printables.com/model/1058668-lwc-meshtastic-meshformer-communicator-node

The " Waypoint": COMMING SOON

**Comment by gargomoma** [comment]
I'll add it to the readme thanks!!

**Comment by lupusworax** [comment]
No, THANK YOU! the faketec is freaking amazing and you are awesome ;)

**Comment by lupusworax** [comment]
The "Waypoint" : https://www.printables.com/model/1104742-meshtastic-waypoint-non-permanent-outdoor-solar-as

**Comment by lupusworax** [comment]
And one more for Team Faketec ;)
https://www.printables.com/model/1116459-meshtastic-meshtak-tacticool-big-boi-water-resista

**Comment by lupusworax** [comment]
And one more design of mine: 
https://www.printables.com/model/1123408-meshtastic-faketec-f1-micro-node-nrf52

**Comment by lupusworax** [comment]
And another one: 
https://www.printables.com/model/1144370-meshtastic-faketak-scout-ultra-portable-tacticool

**Comment by lupusworax** [comment]
Update on my Fakecap Node with a "special edition" with Werwolf tribal imprint and vibra motor update: 

![Image](https://github.com/user-attachments/assets/1c45da09-b12b-488f-9bfe-066030a9254e)

![Image](https://github.com/user-attachments/assets/7a97d3a1-58c0-482e-969e-edd2253f5185)

**Comment by lupusworax** [comment]
Are you powering the gps module permanently or with a mosfet/ transistor? If you use a transistor/mosfet which one exaclty is it?  To take stress off the 3.3 volt rail I connect my GPS + directly  to RAW as the module can take up to 5 volts. (it still is switched off with a mosfet disconnecting it from ground)  Does the fake Tec recognize the gps module but it keeps searching for a fix? 
I also encountered an issue with the fake tec that if you wrongly connected  TX and RX it does not help to change the pins in the app, I always have to physically swap RX/ TX either on the NRF dev board or on the gps module.

**Comment by lupusworax** [comment]
> After fixing it should at least blink its led. I didn't mess anything with wiring, I used this module standalone. I thought it's 3,3V only, even confirmed it on the internet. Are you sure it is 5V tolerant? Anyway, with my faketek's 3,3V LDO it won't work. Maybe it's just doa. Thanks!

RAW does not forward the USBs 5 volts but only the battery directly ( so about 4.2 volts max) I learned this the hard way when I was also pulling my hair as GPS would not power on over USB  with no battery attached XD

**Comment by oldmanav** [comment]
Daft question, apologies if this is listed somewhere and I've not spotted it. How did you print the Tamameshi case? Did you use supports and in what orientation please?

**Comment by oldmanav** [comment]
Solved, print standing up, with tree supports. See the below screenshot from Cura:

<img width="617" height="560" alt="Image" src="https://github.com/user-attachments/assets/fad7177f-4bcc-4030-84e6-f6e088d287c0" />

**Comment by lupusworax** [comment]
Sorry read your question just now, yep standing upright, your 100% correct.

**Comment by oldmanav** [comment]
Yeah. I first printed it face-down with normal supports - it looked a mess, and I gave up trying to remove most of the support material as it was in places where I couldn't reach.
